src/pages/patientView/mutation/column/TumorColumnFormatter.tsx

```diff
@@ -67,6 +96,23 @@ export default class TumorColumnFormatter {
         }, {} as {[s:string]:boolean});
     }
 
+    public static getProfiledSamplesForGene(entrezGeneId:number, samples:ClinicalDataBySampleId[], sampleToGenePanelId:{[sampleId: string]: string|undefined}, genePanelIdToEntrezGeneIds:{[genePanelId: string]: number[]}) {
+        // For a given gene indicate whether it was profiled in a particular sample
+        return samples.reduce((map, nextSample, currentIndex:number) => {
+            const genePanelId = sampleToGenePanelId[nextSample.id];
+            
+            const wholeGenome = noGenePanelUsed(genePanelId);
+            const isInGenePanel = !wholeGenome && genePanelId && genePanelId in genePanelIdToEntrezGeneIds && genePanelIdToEntrezGeneIds[genePanelId].includes(entrezGeneId);
+
+            if (wholeGenome || isInGenePanel) {
```

Comment:
map[nextSample.id] = wholeGenome || isInGenePanel;

also, would be easier to read if renamed `map` to `sampleIsProfiled`

Reply:
> map[nextSample.id] = wholeGenome || isInGenePanel;

Sorry for making this more compicated that needs be, but this change has been applied in a PR builds on top of this PR. So, please disregard this need for code restucturing here.

>also, would be easier to read if renamed map to sampleIsProfiled

Name was updated to `sampleIsProfiled`.

Reply:
I have updated the boolean logic to:
> sampleIsProfiled[nextSampleId] = wholeGenome || isInGenePanel;